src/overlays/actors/ovl_En_Viewer/z_en_viewer.c

```diff
@@ -15,51 +16,81 @@ void EnViewer_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnViewer_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnViewer_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_01000000;
-extern UNK_TYPE D_0404D4E0;
-extern UNK_TYPE D_06000450;
-extern UNK_TYPE D_060005B4;
-extern UNK_TYPE D_060008A0;
-extern UNK_TYPE D_06000BC8;
-extern UNK_TYPE D_06000F54;
-extern UNK_TYPE D_06001410;
-extern UNK_TYPE D_060014F4;
-extern UNK_TYPE D_06001D28;
-extern UNK_TYPE D_06002574;
-extern UNK_TYPE D_06002650;
-extern UNK_TYPE D_06002928;
-extern UNK_TYPE D_060029CC;
+void EnViewer_GetCutsceneNextPos(EnViewer* this, GlobalContext* globalCtx);
+void func_80B2C8AC(EnViewer* this2, GlobalContext* globalCtx);
+void func_80B2CC1C(GlobalContext* globalCtx, EnViewer* this);
+void func_80B2A570(EnViewer* this, GlobalContext* globalCtx);
+void func_80B2A75C(EnViewer* this, GlobalContext* globalCtx);
+
+// sAnimFuncs
+void func_80B2A300(EnViewer* this, GlobalContext* globalCtx, FlexSkeletonHeader* skeletonHeaderSeg,
+                   AnimationHeader* animationSeg);
+void func_80B2A448(EnViewer* this, GlobalContext* globalCtx, FlexSkeletonHeader* skeletonHeaderSeg,
+                   AnimationHeader* animationSeg);
+void func_80B2A4D8(EnViewer* this, GlobalContext* globalCtx, SkeletonHeader* skeletonHeaderSeg,
+                   AnimationHeader* animationSeg);
+
+// sDrawFuncs
+void func_80B2B4A8(EnViewer* this, GlobalContext* globalCtx);
+void func_80B2B8FC(EnViewer* this, GlobalContext* globalCtx);
+void func_80B2BA38(EnViewer* this, GlobalContext* globalCtx);
+void func_80B2C130(EnViewer* this, GlobalContext* globalCtx);
+
+extern Mtx D_01000000;
+extern Gfx* D_0404D4E0;
```

Comment:
The type for display lists is `Gfx[]`, and since they are arrays, you should ideally omit the `&` when referencing them. Relevant to all the other display lists used in this file.